### ISSUE TITLE
The first test case always fails for Windows CI

### ISSUE THREAD

**Issue by Sadaf-A** [neutralinojs/neutralinojs#1278]
**Describe the bug**
The first testcase always fails for Windows CI. I tried swapping the test case with other test case which previously passed. But it also failed after it became the first test case.

**Expected behavior**
All test cases should pass like they do for other platforms

**Specifications**  
 - OS: Windows
 - Neutralinojs version: 5.2.0
 - Neutralinojs client library version: 5.2.0
 - Neutralinojs CLI version: 11.2.0

*Note: you can enter the `neu version` command to find these details*

**Comment by sahitya-chandra** [comment]
@Sadaf-A can you tell me how can I reproduce the issue ??

**Comment by Sadaf-A** [comment]
> @Sadaf-A can you tell me how can I reproduce the issue ??

It was discovered in the github ci runs which happens after a commit is made in the repo

**Comment by sahitya-chandra** [comment]
Can you show a Screenshot of CI workflow that is failing...

**Comment by Sadaf-A** [comment]
> @Sadaf-A If possible can you show a Screenshot of CI workflow that is failing...
> 

![Screenshot_20241225_203110_GitHub.jpg](https://github.com/user-attachments/assets/a12eb9b1-d026-4643-b86d-25e44de30060)

Here, you can see that the first testcase is failing. 
There is nothing wrong with the particular testcase. Any testcase which is first in windows ci is always failing.

**Comment by sahitya-chandra** [comment]
@Sadaf-A What about your tests locally are they all passing ??

**Comment by sahitya-chandra** [comment]
@Sadaf-A What about the tests locally are they all passing??

**Comment by Sadaf-A** [comment]
> @Sadaf-A What about the tests locally are they all passing??

Yes, they all passed locally when I worked on them. You can try running them if you have windows.
